doc-content/enterprise-only/springboot/creating-self-contained-image-proc.adoc

```diff
@@ -108,9 +121,50 @@ The following example adds two version of the `Evaluation` artifact:
     <plugins>
   <build>
 ----
-. To build the self-contained Spring Boot image, enter the following command in the `service` directory:
+ifdef::PAM[]
+. Optional: if you want to be abe to configure the {KIE_SERVER} to communicate with a {CENTRAL} Monitoring instance, add the following lines to the `pom.xml` file under the `<dependencies>` tag:
```

Comment:
So this gives you the ability to figure in the future if you choose? Or does it actually configure {KIE_SERVER} to communicate with a {CENTRAL} Monitoring instance?

Reply:
The former. If you complete this step, then in the future you can configure the server to communicate, etc. You are not in any way obliged to do so - the server will work perfectly fine without BC Monitoring. But if you do NOT do this thing, then later you can not make this KIE server communicate with BC Monitoring.

Reply:
OK